### ISSUE TITLE
[bug] yum check-update output variance

### ISSUE THREAD

**Issue by albatrossflavour** [albatrossflavour/puppet_os_patching#89]
If yum detects that a kernel has been installed but isn't active, it can output different information : 
```
Security: kernel-3.14.6-200.fc20.x86_64 is an installed security update
Security: kernel-3.14.2-200.fc20.x86_64 is the currently running version
```

Causes the following to appear in the os_patching fact:
```
  package_updates => [
    "Security:",
    "Security:"
  ],
  package_update_count => 2,
  security_package_updates => [
    "Security:",
    "Security:"
  ],
```

Need to put a filter in place to ensure the "Security: " lines are caught.

Thanks to @itgrl for flagging this one and just waiting for some extra debugging before I implement a fix.

**Comment by albatrossflavour** [comment]
Created [a branch](https://github.com/albatrossflavour/puppet_os_patching/tree/bugfix/yum) to cover this work

**Comment by albatrossflavour** [comment]
Fixed in #97
